client/app/scripts/superdesk-archive/module.js

```diff
@@ -80,19 +80,21 @@ define([
          * @param {Object} item
          */
         this.spike = function(item) {
-            return api.update(SPIKE_RESOURCE, item, {state: 'spiked'})
-                .then(function() {
-                    if ($location.search()._id === item._id) {
-                        $location.search('_id', null);
-                    }
-                    return item;
-                }, function(response) {
-                    item.error = response;
-                    if (angular.isDefined(response.data._issues) &&
-                        angular.isDefined(response.data._issues['validator exception'])) {
-                        notify.error(gettext(response.data._issues['validator exception']));
-                    }
-                });
+            if (!item.lock_user) {
```

Comment:
I think that you have to check if the list of selected items can be spiked and enable multi spike/unspike button if the operation is possible.